ca/server.go

```diff
@@ -576,6 +583,7 @@ func (s *Server) UpdateRootCA(ctx context.Context, cluster *api.Cluster) {
 		updatedRootCA, err := NewRootCA(rCA.CACert, signingCert, signingKey, expiry, intermediates)
 		if err != nil {
 			logger.WithError(err).Error("invalid Root CA object in cluster")
+			return err
```

Comment:
Good point, fixed this and updated the comment where this is called in cluster update.